package/rpm/pre_install_section

```diff
@@ -0,0 +1,6 @@
+# Add the "decisionengine" user and group if they do not exist
+getent group decisionengine >/dev/null ||
+    groupadd -r  decisionengine
+getent passwd  decisionengine >/dev/null || \
+    useradd -r -g  decisionengine -d /var/lib/decisionengine \
```

Comment:
Like above, should it be %{_sharedstatedir}/decisionengine to be consistent?

Reply:
Fixed